dev/Dockerfile

```diff
@@ -1,10 +1,13 @@
 ARG IMAGE=skpr/php:8.0-1.x
 FROM ${IMAGE}
+ARG PHP_VERSION=8.0
 
 ARG ARCH=amd64
 
 USER root
 
+RUN apk add php${PHP_VERSION}-xdebug
+
 # https://blog.blackfire.io/alpine-linux-support.html
 RUN PHP_VERSION=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && \
```

Comment:
No need for looking up php version if its now an arg.

Reply:
The formatting is a little different between the 2:

ARG = 8.1
php -r approach = 81

Will strip the `.` using bash.